chai/src/main/java/com/droidconke/chai/typography/ChaiTypography.kt

```diff
@@ -0,0 +1,153 @@
+
+private val ChaiDisplayTextStyle = TextStyle(
+    fontSize = 45.sp,
+    fontWeight = FontWeight.Bold,
+    lineHeightStyle = LineHeightStyle(
+        alignment = LineHeightStyle.Alignment.Center,
+        trim = LineHeightStyle.Trim.None
+    ),
+    lineHeight = 1.25.em
+)
+
+@ChaiPreview
+@Composable
+private fun ChaiDisplayTextPreview(){
+    ChaiTheme {
+        Text(text = DUMMY_CHAI_TEXT, style = ChaiDisplayTextStyle)
+    }
+}
+
+private val ChaiTitleTextStyle = TextStyle(
+    fontSize = 32.sp,
+    fontWeight = FontWeight.Bold,
+    lineHeightStyle = LineHeightStyle(
+        alignment = LineHeightStyle.Alignment.Center,
+        trim = LineHeightStyle.Trim.None
+    ),
+    lineHeight = 1.25.em
+)
+@ChaiPreview
+@Composable
+private fun ChaiTitleTextPreview(){
+    ChaiTheme {
+        Text(text = DUMMY_CHAI_TEXT, style = ChaiTitleTextStyle)
+    }
+}
+
+private val ChaiHeaderTextStyle = TextStyle(
+    fontSize = 22.sp,
+    fontWeight = FontWeight.SemiBold,
+    lineHeightStyle = LineHeightStyle(
+        alignment = LineHeightStyle.Alignment.Center,
+        trim = LineHeightStyle.Trim.None
+    ),
+    lineHeight = 1.2.em,
+)
+@ChaiPreview
+@Composable
+private fun ChaiHeaderTextPreview(){
+    ChaiTheme {
+        Text(text = DUMMY_CHAI_TEXT, style = ChaiHeaderTextStyle)
+    }
+}
+
+private val ChaiParagraphTextStyle = TextStyle(
+    fontSize = 16.sp,
+    fontWeight = FontWeight.Normal,
+    lineHeightStyle = LineHeightStyle(
+        alignment = LineHeightStyle.Alignment.Center,
+        trim = LineHeightStyle.Trim.None
+    ),
+    lineHeight = 1.1.em,
+)
+@ChaiPreview
+@Composable
+private fun ChaiParagraphTextPreview(){
+    ChaiTheme {
+        Text(text = DUMMY_CHAI_TEXT, style = ChaiParagraphTextStyle)
+    }
+}
+
+private val ChaiBodyTextStyle = TextStyle(
+    fontSize = 14.sp,
+    fontWeight = FontWeight.Medium,
+    lineHeightStyle = LineHeightStyle(
+        alignment = LineHeightStyle.Alignment.Center,
+        trim = LineHeightStyle.Trim.None
+    ),
+    lineHeight = 1.1.em,
+)
+@ChaiPreview
+@Composable
+private fun ChaiBodyTextPreview(){
+    ChaiTheme {
+        Text(text = DUMMY_CHAI_TEXT, style = ChaiBodyTextStyle)
+    }
+}
+
+private val ChaiLabelTextStyle = TextStyle(
+    fontSize = 12.sp,
+    fontWeight = FontWeight.Bold,
+    lineHeightStyle = LineHeightStyle(
+        alignment = LineHeightStyle.Alignment.Center,
+        trim = LineHeightStyle.Trim.None
+    ),
+    lineHeight = 1.1.em,
+)
+@ChaiPreview
+@Composable
+private fun ChaiLabelTextPreview(){
+    ChaiTheme {
+        Text(text = DUMMY_CHAI_TEXT, style = ChaiLabelTextStyle)
+    }
+}
+
+data class ChaiTypography(
+    val display: TextStyle = ChaiDisplayTextStyle,
+    val title: TextStyle = ChaiTitleTextStyle,
+    val header: TextStyle = ChaiHeaderTextStyle,
+    val paragraph: TextStyle = ChaiParagraphTextStyle,
+    val body: TextStyle = ChaiBodyTextStyle,

```

Comment:
What's the difference between paragraph and body? Can we just have one as the use case seem like it's the same